index.js

```diff
@@ -1,20 +1,18 @@
 module.exports = (password) => {
-  if (!password) {
-    throw new Error("Password is empty.");
-  }
```

Comment:
I would have put this in a separate commit

Reply:
Noted, but I thought making all the required changes in one commit would be acceptable, as there's not too many of them?